### PR TITLE
fix(autotracing): honor cpuidle sampling interval

### PR DESCRIPTION
**Pull request by zjncs**
## Summary

Make cpuidle event tracing use its configured sampling interval.

## Problem

`newCPUIdle` builds a tracer with `Config.CPUIdle.Interval`, but registers a hard-coded 20-second event interval. Configuration changes therefore do not affect scheduler cadence.

## Changes

- Derive `EventTracingAttr.Interval` from the validated tracer interval.

## Verification

- Base SHA: `$(git rev-parse main)`.
- Confirmed current main stores the configured interval in `cpuIdleTracing.interval` while returning `Interval: 20`.
- `git diff --check` passed.
- Go tests unavailable in this environment because `go` and `gofmt` are not installed; CI should run the project test suite.

## Risk

Low; default cadence remains unchanged when the default configuration is used.

AI-assisted contribution; implementation and verification performed against current main.

### CODE REVIEW

File: core/autotracing/cpuidle.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ func newCPUIdle() (*tracing.EventTracingAttr, error) {
 
 	return &tracing.EventTracingAttr{
 		TracingData: tracer,
-		Interval:    20,
+		Interval:    int(tracer.interval / time.Second),
```
Comment (huhong-web) [inline core/autotracing/cpuidle.go:55]: [P2] [CR-001] No test proves a non-default configured interval reaches EventTracingAttr.Interval. Add the corresponding unit regression.\n\nFix: add the focused regression described above.

**Review by huhong-web — COMMENTED** [review]
## Code Review

One non-blocking finding is included inline: core/autotracing/cpuidle.go:55.

## Architecture & Design
- Verdict: Reasonable with a focused verification improvement.
- Evidence: core/autotracing/cpuidle.go:55 changes a behavioral contract without proving the stated edge case.

## Scope & Focus
- Verdict: Focused
- Primary objective: Honor the configured sampling interval.
- Necessary changes: core/autotracing/cpuidle.go:55 and a focused regression test.
- Out-of-scope changes: None.
- Split plan: Not required.

## Review Convergence
- Mode: full
- Base SHA: ae0badc0a1fed0a1403ff352edd33723aeafaf3e
- Reviewed HEAD: 032886d3f591f6feb0b2b1e8aeaf25578994ff4a
- Previous reviewed HEAD: none
- Scope verdict: Focused
- Coverage: Changed files and direct configuration/test contracts; two independent review perspectives.
- Prior findings: 0 open / 0 fixed / 0 verified / 0 withdrawn
- New findings: 1 FIRST_PASS / NON-BLOCKING
- Late non-blocking findings: None
- Full-review reset reason: Not required
- Completeness declaration: At HEAD 032886d3f591f6feb0b2b1e8aeaf25578994ff4a, all changed human-authored files and assigned dimensions were covered; all known blockers are included.

## Verification
- Build/Lint/Integration: NOT RUN — prohibited by review no-build policy.
- Targeted verification: Not required.

---
*Co-authored by: gpt-5.6-terra*
